tests/torchutils_test.py

```diff
@@ -150,7 +182,7 @@ def test_dkl_gauss():
 
     for d1, d2 in zip(dist1, dist2):
         torch_dkl = distributions.kl.kl_divergence(d1, d2)
-        monte_carlo_dkl = dkl_via_monte_carlo(d1, d2, num_samples=1000)
+        monte_carlo_dkl = dkl_via_monte_carlo(d1, d2, num_samples=5000)
```

Comment:
if torch calls it `kl_divergence`, perhaps we should call it `kl`, `kl_d` or something like this ;)